sdk/translation/azure-ai-translation-nspkg/README.md

```diff
@@ -0,0 +1,16 @@
+# Microsoft Azure SDK for Python
+
+This is the Microsoft Azure ai-translation Services namespace package.
```

Comment:
ohh I have never noticed this.
Is this common for all Python packages? and more for curiosity, do you mind explaining what the purpose of it is?

Reply:
Anything that supports python 2 and extends the `azure` namespace needs a namespace package. We have one for azure, azure.ai (added with text analytics) and now azure.ai.translation.

Reply:
Thank you!